static/src/javascripts/projects/commercial/modules/IpsosTaggingScript.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+import { onConsentChange, getConsentFor } from '@guardian/consent-management-platform';
+import config from 'lib/config';
+
+let dm;
+let DotMetricsObj;
+const IpsosTagging = function () {
+
+    console.log("Ipsos tag fired");
+    window.dm = window.dm ||{ AjaxData:[]};
+    window.dm.AjaxEvent = function(et, d, ssid, ad){
+        dm.AjaxData.push({ et,d,ssid,ad});
+        // eslint-disable-next-line no-unused-expressions
+        window.DotMetricsObj && DotMetricsObj.onAjaxDataUpdate();
+    };
+    const d = document;
```

Comment:
you can use `loadScript` from `@guardian/libs` to do this e.g. https://github.com/guardian/frontend/blob/8acdbb104120470dcdaf1257cc15dcd41963fa45/static/src/javascripts/projects/commercial/modules/comscore.js#L4

Reply:
👍